src/06_domains/cittadini-app/99_variables.tf

```diff
@@ -93,3 +93,7 @@ variable "argocd_server_addr" {
   description = "ArgoCD hostname"
 }
 
+variable "arc_alert_enabled" {
```

Comment:
in UAT  environment configuration this should be true (src/06_domains/cittadini-app/env/itn-uat/terraform.tfvars)